packages/core/src/components/table/table-header-cell/table-header-cell.tsx

```diff
@@ -42,7 +42,7 @@ export class TdsTableHeaderCell {
 
   @State() textAlignState: string;
 
-  @State() sortingDirection: 'asc' | 'desc' = 'asc';
+  @State() sortingDirection: 'asc' | 'desc' = 'desc';
```

Comment:
Breaking?

Reply:
Yea, good catch! 
I believe this change should be excluded from the PR if we don't want the PR hanging around until a possible future 2.0 version.

We could instead introduce a prop for the user to change the default sorting direction, for example.

Reply:
I changed this as I felt the current behaviour felt unintended and confusing. Currently on first click after render the sorting goes to descending. But when clicking for the first time on another column it goes to ascending. This only changes the behaviour for the first ever sort click. 

I would also find it strange if this causes braking changes since it's only changing the order of the events. Would only be a braking change if someone has tightly coupled their logic with the first event being dispatched being descending. 

From a feature standpoint I could see it being even better if you could set value yourself in the case that you have a default sorting direction on render. If I am not wrong there is currently no way to load the table with a sorting arrow already visible.

Reply:
@alfredskeden @theJohnnyMe We should look into this.